packages/firestore/lite/index.node.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import firebase from '@firebase/app';
+import { FirebaseNamespace } from '@firebase/app-types';
```

Comment:
Are you planning to make Firestore lite work with the existing SDK? If not, you should take dependency on `@firebase/app-next` and `@firebase/app-types-next` instead.

Reply:
Using `@firebase/app-next` to register components:
```
import { _registerComponent } from '@firebase/app-next';

_registerComponent(new Component(...))
```